From 9f325f9614a1359abb0cbb0b64878d4a4c05e88d Mon Sep 17 00:00:00 2001
From: Simeon Marijon <simeon.marijon@foss.st.com>
Date: Tue, 20 Jun 2023 14:37:01 +0200
Subject: [PATCH 0447/1141] ARM: configs: add stm32 backup registers reboot
 mode support in multiv7 addons fragment

Enable REBOOT MODE from nvmmem cells with
fragment-02-multiv7_addons.config (STM32MP13 && STM32MP15 Socs)

Change-Id: I57a73cceedaf575837005aafd226ba0c09c32012
Signed-off-by: Simeon Marijon <simeon.marijon@foss.st.com>
---
 arch/arm/configs/fragment-02-multiv7_addons.config | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/configs/fragment-02-multiv7_addons.config b/arch/arm/configs/fragment-02-multiv7_addons.config
index 9ff491a4f13b..c3650a026396 100644
--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -47,6 +47,7 @@ CONFIG_MTD_MCHP23K256=m
 CONFIG_MTD_SPI_NAND=y
 CONFIG_NAMESPACES=y
 CONFIG_NR_CPUS=2
+CONFIG_NVMEM_REBOOT_MODE=y
 CONFIG_NVMEM_STM32_ROMEM=y
 CONFIG_NVMEM_STM32_TAMP=y
 CONFIG_OF_OVERLAY=y
-- 
2.39.2

